tensorboard/components/plugin_util/message.ts

```diff
@@ -27,9 +32,10 @@ export type PayloadType =
 
 export interface Message {
   type: string;
-  id: string;
+  id: number;
   payload: PayloadType;
   error: string | null;
+  isReply: boolean;
```

Comment:
Is this for readability of the module? Or does it have functional differences?

Reply:
Functional difference, and the test I added covers it!

Basically, if a frame receives message `{id: 0, ...}`, it doesn't know whether it's the reply-message to something it previously sent, or whether it's a brand new incoming message from another frame.

In both cases, the origin is a foreign frame, but we need to distinguish that the former case should resolve the responseWait, while the latter should not.